Formula/nerd-fonts-patcher.rb

```diff
@@ -0,0 +1,28 @@
+class NerdFontsPatcher < Formula
+  desc "Patch your own font"
+  homepage "https://github.com/ryanoasis/nerd-fonts#font-patcher"
+  url "https://github.com/ryanoasis/nerd-fonts/trunk/src", using: :svn
+  version = version "2.1.0"
```

Comment:
```suggestion
  version "2.1.0"
```

Reply:
@SMillerDev This was just so I could refer to `version` inside the `resource` block… There doesn't seem to be accessible `instance_variables` in Formulae, like there are in Casks (e.g `token`, `@cask.methods` etc)… why is this?